cdap-support-bundle/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -105,4 +139,130 @@ public void createSupportBundle(HttpRequest request, HttpResponder responder,
+    }
+    List<SupportBundleOperationStatus> bundleOperationStatusList = new ArrayList<>();
+    List<File> supportFiles = DirUtils.listFiles(baseDirectory)
+      .stream()
+      .filter(file -> !file.getName().startsWith(".") && !file.isHidden() && file.isDirectory())
+      .collect(Collectors.toList());
+    for (File uuidFile : supportFiles) {
+      SupportBundleOperationStatus bundleOperationStatus =
+        bundleGenerator.getSingleBundleJson(uuidFile.getName());
+      bundleOperationStatusList.add(bundleOperationStatus);
+    }
+    responder.sendString(HttpResponseStatus.OK, GSON.toJson(bundleOperationStatusList));
+  }
+
+  /**
+   * Get the specific bundle status from status.json by using uuid.
+   *
+   * @param uuid the bundle id which is also the uuid
+   * @throws NotFoundException is thrown if status.json is not found
+   */
+  @GET
+  @Path("/support/bundles/{uuid}/status")
+  public void getSupportBundleByUUID(HttpRequest request, HttpResponder responder, @PathParam("uuid") String uuid)
+    throws Exception {
+    SupportBundleEntityId bundleEntityId = new SupportBundleEntityId(uuid);
+    contextAccessEnforcer.enforce(bundleEntityId, StandardPermission.GET);
+    List<SupportBundleOperationStatus> bundleOperationStatusList = new ArrayList<>();
+    SupportBundleOperationStatus bundleOperationStatus = bundleGenerator.getSingleBundleJson(uuid);
+    bundleOperationStatusList.add(bundleOperationStatus);
+    responder.sendString(HttpResponseStatus.OK, GSON.toJson(bundleOperationStatusList));
+  }
+
+  /**
+   * Delete the specific bundle by using the uuid
+   *
+   * @param uuid the bundle id which is also the uuid
+   * @throws NotFoundException is thrown if status.json is not found
+   */
+  @DELETE
+  @Path("/support/bundles/{uuid}")
+  public void deleteSupportBundle(HttpRequest request, HttpResponder responder, @PathParam("uuid") String uuid)
+    throws Exception {
+    SupportBundleEntityId bundleEntityId = new SupportBundleEntityId(uuid);
+    contextAccessEnforcer.enforce(bundleEntityId, StandardPermission.DELETE);
+    File baseDirectory = new File(cConf.get(Constants.SupportBundle.LOCAL_DATA_DIR));
+    File uuidFile = new File(baseDirectory, uuid);
+    if (!baseDirectory.exists()) {
+      throw new NotFoundException("No content in Support Bundle.");
+    }
+    if (!uuidFile.exists()) {
+      throw new NotFoundException(String.format("No such uuid '%s' in Support Bundle.", uuid));
+    }
+    bundleGenerator.deleteOldFoldersIfExceedLimit(uuidFile);
+    responder.sendString(HttpResponseStatus.OK, String.format("Successfully deleted bundle %s", uuid));
+  }
+
+  /**
+   * Exports all selected files as a ZIP archive file.
+   *
+   * @param uuid the bundle id which is also the uuid fileJson body
+   */
+  @POST
+  @Path("/support/bundle/{uuid}/download")
+  public void appsExport(FullHttpRequest request, HttpResponder responder, @PathParam("uuid") String uuid)
```

Comment:
Why call it appExport? Should be call `exportSupportBundle` or `downloadSupportBundle`.